app/controllers/org_admin/templates_controller.rb

```diff
@@ -0,0 +1,450 @@
+module OrgAdmin
+  class TemplatesController < ApplicationController
+    include Paginable
+    after_action :verify_authorized
+
+    # GET /org_admin/templates
+    # -----------------------------------------------------
+    def index
+      authorize Template
+      valid_orgs = current_user.can_super_admin? ? Org.non_funders : Org.find(current_user.org)
+      org_templates = get_latest_template_versions(valid_orgs).page(1)
+      funder_templates = get_latest_template_versions(Org.funders).page(1)
+
+      # If the user is an Org Admin look for customizations to funder templates
+      customizations = []
+      unless current_user.can_super_admin?
```

Comment:
Not at this point. I am going to open a separate ticket to revisit and add the additional column that contains the customization controls/status. I'd like to get this merged asap since the template objects have been moved to org_admin namespace.

Reply:
ok, that sounds sensible to handle in another ticket.